guides/common/modules/proc_configuring-how-many-processes-can-be-launched-by-apache-httpd.adoc

```diff
@@ -0,0 +1,25 @@
+[id="configuring_how_many_processes_can_be_launched_by_apache_httpd_{context}"]
+= Configuring how many processes can be launched by Apache httpd
```

Comment:
Could we rephrase the title to something like "Tuning Apache httpd child processes"?
Note that the ID and the filename need also to be changed.

Reply:
Done. Thanks for the suggestion.